css/src/atomics/border.scss

```diff
@@ -0,0 +1,55 @@
+.border {
+	border: $border-width solid $border !important;
+}
+
+.border-none {
+	border: 0 !important;
+}
+
+@include tablet {
+	.border-tablet {
+		border: $border-width solid $border !important;
+	}
+
+	.border-none-tablet {
+		border: 0 !important;
+	}
+}
+
+// Rules for each side
+@each $border-key, $direction in $directions {
+	.border-#{$border-key} {
+		border-#{$direction}: $border-width solid $border !important;
+	}
+
+	.border-#{$border-key}-none {
+		border-#{$direction}: 0 !important;
+	}
+
+	@include tablet {
```

Comment:
```scss

// unnest 👇?
@include tablet {
  @each $border-key, $direction in $directions {
    		.border-#{$border-key}-tablet {
			border-#{$direction}: $border-width solid $border !important;
		}

		.border-#{$border-key}-none-tablet {
			border-#{$direction}: 0 !important;
		}
  }
}

Reply:
See resolve conversation at the top of the file.

Reply:
updated